Library_File_maneger_NetFramework/Commands/ChangeDir.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.IO;
+
+using Library_File_Maneger.Commands.Bace;
+
+namespace Library_File_Maneger.Commands
+{
+    internal sealed class ChangeDir : FileManagerCommand
+    {
+        public override DataDirectores[] DataDirs { get; set; }
+
+        public ChangeDir(DataDirectores[] dataDir)
+        {
+            DataDirs = dataDir;
+        }
+
+        public override void Execute(string[] args)
+        {
+            if (Directory.Exists(args.Last()))
```

Comment:
Ну зачем использовать `Linq` там, где в этом нет никакой необходимости?
```suggestion
            if (Directory.Exists(args[^1]))
```

Reply:
```suggestion
            if (Directory.Exists(args[args.Length - 1]))
```

Reply:
Linq - очень тормозной!